cnf-tests/testsuites/e2esuite/security/tuning_sriov.go

```diff
@@ -35,12 +31,10 @@ var _ = Describe("[sriov] Tuning CNI integration", func() {
 	apiclient := client.New("")
 
 	execute.BeforeAll(func() {
-		err := namespaces.Create(SriovTestNamespace, apiclient)
+		err := namespaces.Create(namespaces.SriovTuningTest, apiclient)
 		Expect(err).ToNot(HaveOccurred())
-	})
 
-	AfterEach(func() {
-		namespaces.CleanPods(SriovTestNamespace, apiclient)
+		namespaces.CleanPods(namespaces.SriovTuningTest, apiclient)
```

Comment:
we have this also on line 48 so we maybe we can remove it from here or the other way around?

Reply:
Good point! As there are two test cases that create pods, I'll move it to the BeforeEach section